doc/man1/flux-kvs.rst

```diff
@@ -65,170 +65,616 @@ setting the namespace in the environment variable :envvar:`FLUX_KVS_NAMESPACE`.
+  or the value of :envvar:`FLUX_KVS_NAMESPACE` is used.
+
+move
+----
+
+.. program:: flux kvs move
+
+Copy *source* key to *destination* key and unlink *source*.
+
+.. option:: -h, --help
+
+  Display help on this sub-command.
+
+.. option:: -S, --src-namespace=NAME
+
+  Specify the source namespace.  By default, the primary KVS namespace
+  or the value of :envvar:`FLUX_KVS_NAMESPACE` is used.
+
+.. option:: -D, --dst-namespace=NAME
+
+  Specify the destination namespace.  By default, the primary KVS namespace
+  or the value of :envvar:`FLUX_KVS_NAMESPACE` is used.
+
+getroot
+-------
+
+.. program:: flux kvs getroot
+
+Retrieve the current KVS root directory reference, displaying it as an
+RFC 11 dirref object unless otherwise specified.
+
+.. option:: -h, --help
+
+  Display help on this sub-command.
+
+.. option:: -N, --namespace=NAME
+
+  Specify an alternate namespace.  By default, the primary KVS namespace
+  or the value of :envvar:`FLUX_KVS_NAMESPACE` is used.
+
+.. option:: -o, --owner
+
+   Display the numerical user ID that owns the target namespace.
+
+.. option:: -b, --blobref
+
+   Display the root directory reference as a single blobref.
+
+.. option:: -s, --sequence
+
+   Display the root sequence number or "version".
+
+version
+-------
+
+.. program:: flux kvs version
+
+Display the current KVS version, an integer value. The version starts
+at zero and is incremented on each KVS commit. Note that some commits
+may be aggregated for performance and the version will be incremented
+once for the aggregation, so it cannot be used as a direct count of
+commit requests.
+
+.. option:: -h, --help
+
+  Display help on this sub-command.
+
+.. option:: -N, --namespace=NAME
+
+  Specify an alternate namespace.  By default, the primary KVS namespace
+  or the value of :envvar:`FLUX_KVS_NAMESPACE` is used.
+
+wait
+----
+
+.. program:: flux kvs wait
+
+Block until the KVS version reaches *version* or greater. A simple form
+of synchronization between peers is: node A puts a value, commits it,
+reads version, sends version to node B. Node B waits for version, gets value.
+
+.. option:: -h, --help
+
+  Display help on this sub-command.
+
+.. option:: -N, --namespace=NAME
+
+  Specify an alternate namespace.  By default, the primary KVS namespace
+  or the value of :envvar:`FLUX_KVS_NAMESPACE` is used.
+
+namespace create
+----------------
+
+.. program:: flux kvs namespace create
+
+Create a new KVS namespace.
+
+.. option:: -h, --help
+
+  Display help on this sub-command.
+
+.. option:: -o, --owner=UID
+
+  Set the owner of the namespace to *UID*.  If unspecified, the owner is
+  set to the Flux instance owner.
+
+.. option:: -r, --rootref=TREEOBJ
+
+  Initialize namespace with specific root directory reference
+  If unspecified, an empty directory is referenced.
+
+namespace remove
+----------------
+
+.. program:: flux kvs namespace remove
+
+Remove a KVS namespace.
+
+.. option:: -h, --help
+
+  Display help on this sub-command.
```

Comment:
perhaps note that remove is done in the background? i.e. may not be removed when command returns.